components/stats/Chart.js

```diff
@@ -204,26 +267,124 @@ export default class Chart extends React.Component {
     }
   }
 
-  render() {
-    if (this.state.organizedData.length) {
+  //Begin line chart render methods
+
+/*   createGrid = () => {
+    const maxX = Date.parse(this.getMaxX());
+    const minX = Date.parse(this.getMinX());
+    const { gridLines } = this.props;
+    const { xWall } = this.props;
+    let diffX = maxX - minX;
+    let dX = xWall / diffX;
+    //let bufferY = (600 - yCeiling) / 2;
+    let bufferX = (600 - xWall) / 2;
+    const beginX = (Math.floor(minX) + bufferX) * dX
+    const endX = (Math.floor(maxX) - bufferX) * dX
+    let spacing = (beginX - endX)/gridLines;
+    const gLines = [];
+
+    for(let gridX = beginX; gridX > endX; gridX+= spacing){
+      gLines.push(gridX);
+      console.log(gridX);
+    }
+
+    return null;
+  } */
+
+  createCircles = () => {
+    const organizedData = this.state.organizedData;
+    let oData = organizedData.flat(2);
+    let allCircles = oData.map((g, index) => {
       return (
-        <React.Fragment>
-          <CreateChart
-            minX={this.state.minX}
-            maxX={this.state.maxX}
-            minY={this.state.minY}
-            maxY={this.state.maxY}
-            deltaX={this.state.deltaX}
-            deltaY={this.state.deltaY}
-            ticks={this.state.ticks}
-            xLabel={this.state.xLabel}
-            yLabel={this.state.yLabel}
-            organizedData={this.state.organizedData}
-          />
-        </React.Fragment>
+        <circle key={index} cx={g.x} cy={g.y} r="2" css={STYLES_CHART_CIRCLE} />
+      );
+    });
+    ReactDOM.render(allCircles, document.getElementById("circles"));
```

Comment:
`ReactDOM` isn't necessary, you should be able to get this all within `render` using `map` directly inside of a render function. For example, this is acceptable:

```
render() {
   return (
      <div>{allCircles}</div>
   )
}
```

Reply:
It took some work but I refactored to remove reactDOM from the component and used your suggested method!